src/website/views/components/brand.pug

```diff
@@ -60,10 +60,10 @@ p.-text
 .example.-mb--3
   .-p--3(style='overflow:auto')
     each size in sizes
-      div(class=`a-col -w--12 -p--3`)
+      div(class=`a-col -w--12 -p--2`)
```

Comment:
@davidojedacl also change -p--2 to -pb--2 in all the places where there are sizes examples

Reply:
done